vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion_util.cc

```diff
@@ -246,6 +255,101 @@ std::unique_ptr<base::ListValue> ParseStringToBaseList(
   return std::make_unique<base::ListValue>(value->GetList());
 }
 
+bool UnBlindTokens(
+    ledger::PromotionPtr promotion,
+    std::vector<std::string>* unblinded_encoded_tokens,
+    std::string* error) {
+  if (
+      !promotion ||
+      !promotion->credentials ||
+      !unblinded_encoded_tokens ||
+      !error) {
+    *error = "Data sent in is not correct";
```

Comment:
There's a check for `!error` above but then we set the error here anyways. Wouldn't this cause a crash if we actually did provide a null ptr here?

Reply:
you are right, I forgot to remove this when testing

Reply:
fixed